documentation/testing-docs/test-matrix-for-az-accounts.md

```diff
@@ -0,0 +1,196 @@
+# Test Matrix for Authentication in Az.Accounts
+
+The authentication functionality in Az.Accounts is one of most important piece for Azure PowerShell, to make sure Azure PowerShell be delivered to customers with high quality, we define the test matrix and it should be honored by each release of Az.Accounts.
+
+**Priority Clarification**
+
+- `P0`: Run if any authentication related code change in Az.Accounts
```

Comment:
P0 > P1 > P2 is more align to convention.

Reply:
Yes, that is what I mean, refine description for better understanding.